psi-ms.obo

```diff
@@ -22350,6 +22350,33 @@ name: mzTab-M
 def: "Expanded tabular result format for metabolomics experiments reporting quantitative summary data, MS features and identification evidence." [PMID:30688441, PMID:31525911, https://www.psidev.info/mztab]
 is_a: MS:1000914 ! tab delimited text format
 
+[Term]
+id: MS:1003390
+name: crosslinker cleavage
+def: "Signifies that the crosslinker is cleavable and on cleavage can leave a given stub. The pattern specifies up to three slots <name>:<mass>:<pairs with>." [PSI:PI]
+is_a: MS:1001471 ! peptide modification details
+relationship: has_regexp MS:1003391 ! crosslinker cleavage regular expression
```

Comment:
Do we also need a line of relationship: has_value_type xsd:string here? Or do readers understand that has_regexp implies that? I'm thinking we need to be explicit about its value_type?

Reply:
would it make sense to add
relationship: has_value_type xsd:string
to id: MS:1002479?
```
[Term]
id: MS:1002479
name: regular expression
def: "Regular expression." [PSI:PI]
relationship: part_of MS:0000000 ! Proteomics Standards Initiative Mass Spectrometry Vocabularies
relationship: has_value_type xsd:string
```
then every regex would have that